tensorboard/backend/event_processing/db_import_multiplexer.py

```diff
@@ -171,6 +172,14 @@ def producer():
       del self._run_loaders[loader.subdir]
     tf.logging.info('Finished with DbImportMultiplexer.Reload()')
 
+  def _get_exp_and_run_names(self, path, subdir, experiment_name_override=None):
+    if experiment_name_override is not None:
+      return (experiment_name_override, os.path.relpath(subdir, path))
+    sep = io_wrapper.PathSeparator(path)
+    path_parts = os.path.relpath(subdir, path).split(sep)
```

Comment:
Prefer split(sep, 1) to avoid unnecessary split and rejoin

```python
path_parts = os.path.relpath(subdir, path).split(sep, 1)
experiment_name = path_parts[0]
run_name = path_parts[1:] or '.'
```